tcms/xmlrpc/api/testcase.py

```diff
@@ -286,18 +286,19 @@ def create(values, **kwargs):
 
         # Add tag to the case
         for tag in string_to_list(values.get('tag', [])):
-            t, c = Tag.objects.get_or_create(name=tag)
-            tc.add_tag(tag=t)
+            tag, _ = Tag.objects.get_or_create(name=tag)
+            test_case.add_tag(tag=tag)
     else:
         # Print the errors if the form is not passed validation.
         raise ValueError(form_errors_to_list(form))
 
-    result = tc.serialize()
-    result['text'] = tc.latest_text().serialize()
+    result = test_case.serialize()
+    result['text'] = test_case.latest_text().serialize()
 
     return result
 
 
+# pylint: disable=redefined-builtin
```

Comment:
I think this is better to be placed after the function name. However you have to do little experimenting.

From:
https://docs.pylint.org/en/1.8/faq.html#is-it-possible-to-locally-disable-a-particular-message
```at the desired block level or at the end of the desired line of code```. The way I understand it is that the disable placed above the function name will turn off the redefined-builtin check until the end of the module, not only for that function.

A simple way to test this is to deliberately introduce another function which will raise warnings (even the same warning) and see which positioning kills warnings for both functions. We still want to disable checks very cautiously.

Reply:
Here are the results of my experiment:
I left `#pylint: disable=redefined-builtin` on top of the function signature as it is here, and defined two methods with names that shadow builtin function below that - and pylint ignored them

However, when I put `#pylint: disable=redefined-builtin` next to the function signature it caught the two other below, so it is safe to put in next to the function definition, e.g. 
`def filter(query): # pylint: disable=redefined-builtin`